### ISSUE TITLE
python completer is barfing on attribute completion

### ISSUE THREAD

**Issue by gforsyth** [xonsh/xonsh#2173]
```python
gil@theo ~/git/xonsh master 🐚  env XONSH_DEBUG=1 xonsh --no-rc
gil@theo ~/git/xonsh master $ import numpy
gil@theo ~/git/xonsh master $ numpy.zeros(<TAB>
Exception in thread Thread-31:
Traceback (most recent call last):                                                                      
  File "/home/gil/miniconda3/lib/python3.5/threading.py", line 914, in _bootstrap_inner
    self.run()
  File "/home/gil/miniconda3/lib/python3.5/threading.py", line 862, in run
    self._target(*self._args, **self._kwargs)
  File "/home/gil/miniconda3/lib/python3.5/site-packages/prompt_toolkit/interface.py", line 840, in run
    completions = list(buffer.completer.get_completions(document, complete_event))
  File "/home/gil/miniconda3/lib/python3.5/site-packages/xonsh/ptk/completer.py", line 42, in get_compls
    self.ctx)
  File "/home/gil/miniconda3/lib/python3.5/site-packages/xonsh/completer.py", line 35, in complete
    out = func(prefix, line, begidx, endidx, ctx)
  File "/home/gil/miniconda3/lib/python3.5/site-packages/xonsh/completers/base.py", line 20, in complete
    python_comps = complete_python(prefix, line, start, end, ctx)
  File "/home/gil/miniconda3/lib/python3.5/site-packages/xonsh/completers/python.py", line 53, in compln
    rtn = _complete_python(prefix, line, start, end, ctx)
  File "/home/gil/miniconda3/lib/python3.5/site-packages/xonsh/completers/python.py", line 77, in _compn
    args = python_signature_complete(prefix, line, end, ctx, filt)
  File "/home/gil/miniconda3/lib/python3.5/site-packages/xonsh/completers/python.py", line 174, in pythe
    sig = inspect.signature(val)
  File "/home/gil/miniconda3/lib/python3.5/inspect.py", line 2987, in signature
    return Signature.from_callable(obj, follow_wrapped=follow_wrapped)
  File "/home/gil/miniconda3/lib/python3.5/inspect.py", line 2737, in from_callable
    follow_wrapper_chains=follow_wrapped)
  File "/home/gil/miniconda3/lib/python3.5/inspect.py", line 2230, in _signature_from_callable
    skip_bound_arg=skip_bound_arg)
  File "/home/gil/miniconda3/lib/python3.5/inspect.py", line 2060, in _signature_from_builtin
    raise ValueError("no signature found for builtin {!r}".format(func))
ValueError: no signature found for builtin <built-in function zeros>
```

**Comment by olethanh** [comment]
I also have this problem with print
```
print(<tab>
ValueError: no signature found for builtin <built-in function print>

```

**Comment by gforsyth** [comment]
Fixed in #2174
